src/epcpm/cantoxlsx.py

```diff
@@ -454,24 +437,36 @@ def format_for_manual(
         entered_tables_section = False
 
         for row in filtered_rows:
-            parameter_path = row[16].value
+            is_numbered_variant = False
+
```

Comment:
I don't think this is necessary since `is_numbered_variant` is set below every time.
```suggestion
```

Reply:
[d54a0f7](https://github.com/epcpower/pm/pull/91/commits/d54a0f7ec0b9deded82cea8d541b3781aaaa7e2e)